doc/modules/ensemble.rst

```diff
@@ -150,6 +150,16 @@ In contrast to the original publication [B2001]_, the scikit-learn
 implementation combines classifiers by averaging their probabilistic
 prediction, instead of letting each classifier vote for a single class.
 
+Comparison of Histogram-based Gradient Boosting Trees over Random Forests in terms of computational cost:
+
+   - Building shallow trees: HGBT can be computationally more efficient than RF when building shallow trees because it only needs to consider a limited number of bins to construct the splits, whereas RF builds deep trees that require more iterations to reach optimal splits.
+
+   - Sequential boosting: In HGBT, the trees are built sequentially, with each tree correcting the mistakes of the previous trees. This can be more computationally efficient than RF, where all the trees are built independently and in parallel.
+
+   - Efficient binning: HGBT uses an efficient binning algorithm that can handle large datasets with a high number of features. The binning algorithm can pre-process the data to speed up the subsequent tree construction. In contrast, RF does not use binning and relies on randomly selecting features at each split, which can be computationally expensive.
```

Comment:
Here `XXX` is to be filled by you with a reference to the [Why it's faster section](https://scikit-learn.org/stable/modules/ensemble.html#why-it-s-faster).

```suggestion
- Efficient binning: HGBT uses an efficient binning algorithm that can handle
  large datasets with a high number of features. The binning algorithm can
  pre-process the data to speed up the subsequent tree construction (see
  subsection XXX). In contrast, the scikit-learn implementation of random
  forests does not use binning and relies on exact splitting, which can be
  computationally expensive.
```

Reply:
> Here `XXX` is to be filled by you with a reference to the [Why it's faster section](https://scikit-learn.org/stable/modules/ensemble.html#why-it-s-faster).

I have applied the changes. Please review